test/libsolidity/syntaxTests/visibility/interface/interface_contract_function_default.sol

```diff
@@ -0,0 +1,10 @@
+interface I {
+    function f();
+}
+contract C {
+    function g();
+}
+// ----
+// SyntaxError: (18-31): No visibility specified. Did you intend to add "external"?
+// SyntaxError: (51-64): No visibility specified. Did you intend to add "public"?
```

Comment:
I think this test case shouldn't be placed here in the `interface` directory. Also there is a duplicate test case for the interface part: `syntaxTests/visibility/interface/function_default.sol`

I'd create a test case for the `contract` case and leave it where the contract tests are.

Reply:
Ok, perhaps leave that as a comment in the file? Otherwise it could be deleted in the future as it is hard to figure out the reasoning behind it.

Reply:
Sounds good. I've added the comment.